examples/hello-c-client/main.c

```diff
@@ -1,7 +1,7 @@
 #include <launchdarkly/client_side/bindings/c/sdk.h>
 
-#include <launchdarkly/bindings/c/config/builder.h>
 #include <launchdarkly/bindings/c/context_builder.h>
+#include <launchdarkly/client_side/bindings/c/config/builder.h>
 
```

Comment:
This demonstrates the customer change necessary: append `client_side` to the left of `/bindings`.